brainstormforce/astra
# AST-1578 - Pencil edit icon for sticky add to cart and shop filter

### Description
This Pr fixes 
https://brainstormforce.atlassian.net/browse/AST-2106
https://brainstormforce.atlassian.net/browse/AST-2106

### Screenshots
https://d.pr/i/TE6knv
https://d.pr/xXcHnz

### Types of changes
New feature (non-breaking change which adds functionality) 

### How has this been tested?
- Check in customizer
- Check if the edit icon takes you too the correct section / control 

### Checklist:
